usagov/vote-gov-drupal
# VOTE-2902: Add state specific nvrf api data endpoint.

## Jira ticket

[VOTE-2902](https://cm-jira.usa.gov/browse/VOTE-2902)

## Description

Update NVRF API view to include a dynamic state specific data view.

## Deployment and testing

### Post-deploy steps

1. Locally, run `lando retune` to install updates.

### QA/Testing instructions

1. Goto /nvrf/assets/states/{state_abbreviation}/data.json
2. Replace {state_abbreviation} with a state (ie. nj) and confirm that the correct json data is loaded.
3. Test the same url with translation /{langcode}/nvrf/assets/states/{state_abbreviation}/data.json
4. Repeat step 2

## Checklist for the Developer




## Checklist for the Peer Reviewers
